modules/genomic_browser/ajax/genomic_file_upload.php

```diff
@@ -107,6 +114,46 @@ function validateRequest()
 }
 
 /**
+ * Sets $fileToUpload->full_path
+ *
+ * @param stdClass $fileToUpload The file to upload
+ *
+ * @return void
+ */
+function setFullPath(&$fileToUpload)
+{
+    $config           = NDB_Config::singleton();
+    $genomic_data_dir = rtrim($config->getSetting('GenomicDataPath'), '/')
+        . "/genomic_uploader/";
```

Comment:
creating /data/genomics/genomic_uploader, and chown group to www-data allows file to be uploaded and inserted successfully, and the file appears in the front end. Unable to test with genomics files since I don't have any.